### PR TITLE
Mobile Menu of portofolio

### PR DESCRIPTION
**Pull request by SabaAhmad404**
This is a website portfolio In this project I worked on
1) add the linters
2) Add HTML and CSS file
3) make the mobile menu using Javascript 
Kindly review my project

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -35,21 +36,37 @@ body {
   font-size: 32px;
   line-height: 44px;
   color: white;
+  background-color: rgba(96, 112, 255, 0.7);
```
Comment (tufoinnkuo10) [inline style.css:39]: - [ ] kindly use the correct background color of `#6070FF;`

File: index.html [hunk 2]
```diff
@@ -29,8 +29,18 @@
...
         </div>
-      </nav>
-    </header>
+      </nav> 
+    </header> 
+    <div class="mobile-menu-list">
+      <div class="closeImg">
+      <img class="pop-up-close-btn" src="./images/cross.png"  alt="cross">
+      </div>
+      <ul>
```
Comment (tufoinnkuo10) [inline index.html:38]: - [ ] please follow the figma design, padding, font size 
- [ ] follow the figma close icon position as well

yours
Screenshot (536)_LI

figma
Screenshot (535)

File: index.html [hunk 3]
```diff
@@ -29,8 +29,18 @@
...
+        <li class="MM-list"><a class="mobile-menu-link for-clik-1" href="#" >Portfolio</a></li>
+        <li class="MM-list"><a class="mobile-menu-link for-clik-2" href="#" >About</a></li>
+        <li class="MM-list"><a class="mobile-menu-link for-clik-3" href="#">Contact</a></li>
+      </ul>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please ensure that onclick of the menu links, the popup menu disappears and it takes you to the right page section. great work

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
## Hi @SabaAhmad786 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile menu hamburger works as expected
✅ Menu close icon works perfectly well

## Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by dasileker — APPROVED** [review]
### **Hi @SabaAhmad786,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**